nameko/contrib/sqlalchemy.py

```diff
@@ -3,12 +3,12 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 
-from nameko.extensions import Dependency
+from nameko.extensions import DependencyProvider
 
 ORM_DB_URIS_KEY = 'ORM_DB_URIS'
 
 
-class OrmSession(Dependency):
+class OrmSession(DependencyProvider):
```

Comment:
Yep this is confusing. It's about to get moved out into a separate codebase as a "community extension". Hopefully that will also help clarify the whole extension/entrypoint/dependency confusion.